java/src/jmri/jmrit/symbolicprog/VariableTableModel.java

```diff
@@ -1079,9 +1079,13 @@ public boolean decoderDirty() {
     public VariableValue findVar(String name) {
         for (int i = 0; i < getRowCount(); i++) {
             if (name.equals(getItem(i))) {
+            log.debug("findVar matched '{}' by Item", name);
                 return getVariable(i);
             }
```

Comment:
The indentation is wrong for the log statement, making the code more difficult to read.